<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubMenu</class>
 <widget class="QWidget" name="SubMenu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>321</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linea_nombre">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Apellido</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linea_apellido"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="dni">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DNI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linea_dni"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefono</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linea_telefono"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Direccion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linea_direccion"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="boton_guardar">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_guardar_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>linea_nombre</zorder>
  <zorder>linea_apellido</zorder>
  <zorder>label</zorder>
  <zorder>text</zorder>
  <zorder>label_3</zorder>
  <zorder>dni</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>linea_dni</zorder>
  <zorder>linea_telefono</zorder>
  <zorder>linea_direccion</zorder>
  <zorder>boton_guardar</zorder>
  <zorder></zorder>
  <zorder>text</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_guardar_2</sender>
   <signal>clicked()</signal>
   <receiver>SubMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
